src/test/java/JSONSerializerTest.java

```diff
@@ -0,0 +1,58 @@
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class JSONSerializerTest {
+    private JacksonJSONSerializer serializer;
+
+    @BeforeEach
+    void setUp() {
+        final ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        serializer = new JacksonJSONSerializer(objectMapper);
+    }
+
+    @Test
+    void serialize() {
+        Map<String, Integer> values = createInputMap("gameId", 456);
+        String expected = "{\"gameId456\":456}";
+
+        Optional<String> actual = serializer.serialize(values);
+
+        assertThat(actual).hasValue(expected);
+    }
+
+    @Test
+    void getEmptyStringWhenSerializingNull() {
+        Optional<String> actual = serializer.serialize(null);
+
+        assertFalse(actual.isPresent());
+    }
+
+    @Test
+    void doNotSerializeNullValues() {
+        Map<String, Integer> values = createInputMap("gameId", 456, null);
+
+        String expected = "{\"gameId456\":456}";
+
+        Optional<String> actual = serializer.serialize(values);
+
+        assertThat(actual).hasValue(expected);
+    }
+
+    private Map<String, Integer> createInputMap(String name, Integer... values) {
```

Comment:
nu ir dar kartą paminėsiu, kad IDE dešiniam viršutiniam kampe yra mažas kvadračiukas. Kol jis neatrodo taip: 
![image](https://user-images.githubusercontent.com/12122588/74711911-bf04f300-522d-11ea-8564-9633b624f344.png), tol tvarkai tą failą. tau šitoj vietoj degė oranžine spalva, nes name visada tas pats ir užtektų alt+enter paspaust ir tau pats IDE pasiūlytų inline padaryt šitą paramą